quickwit/quickwit-cli/src/cli.rs

```diff
@@ -35,7 +35,8 @@ pub fn build_cli<'a>() -> Command<'a> {
                 .help("Config file location")
                 .env("QW_CONFIG")
                 .default_value(DEFAULT_QW_CONFIG_PATH)
-                .global(true),
+                .global(true)
+                .display_order(0),
```

Comment:
+1

Reply:
@guilload I just fixed the `display_order` but there is another trouble with the tests.
The option `data_dir` that I have deleted is required for tests so I removed my last commit.